File: result/translate/starchat/python2java/java/COUNT_NUMBER_PAIRS_N_B_N_GCD_B_B.java
public static int f_gold ( int n ) {
        int k = n, imin = 1, ans = 0;
        while ( imin <= n ) {
            int imax = (int) ( n / k );
            ans += k * ( imax - imin + 1 );
            imin = imax + 1;
            k = (int) ( n / imin );
        }
        return ans;
    }

File: dataset/python2java/python/COUNT_NUMBER_PAIRS_N_B_N_GCD_B_B.py
def f_gold(n):
    k = n
    imin = 1
    ans = 0
    while ( imin <= n ) :
        imax = int ( n / k )
        ans += k * ( imax - imin + 1 )
        imin = imax + 1
        k = int ( n / imin )
    return ans